style.css

```diff
@@ -56,6 +56,7 @@ body{
 
 body{
     font-family: 'Hind Siliguri', sans-serif;
+    overflow-x: hidden;
```

Comment:
@PrattDarsh Please remove this line and check for possible CSS errors elsewhere.